src/edges_analysis/analysis/levels.py

```diff
@@ -1607,7 +1607,7 @@ def get_s11_paths(
                 fls.append(p)
 
             return fls
-        if s11_path.endswith("csv"):
+        if (os.path.isfile(s11_path)) and (not s11_path.endswith("s1p")):
```

Comment:
@steven-murray when I was writing this I realized that s11_path.is_dir() won't work.

Reply:
so I am confused how didnt we get an error in line 1618 (s11_path.is_dir()) ?

Reply:
You'll need to make sure you convert the `s11_path` to a `pathlib.Path` instance (i.e. do `s11_path = Path(s11_path)`) before you can use the `is_dir` method on it

Reply:
But I can accept this as is.